cmd/statusd/main.go

```diff
@@ -57,6 +62,24 @@ func main() {
 	// wait till node is started
 	<-started
 
+	// Check if debugging CLI connection shall be enabled.
+	if *cli {
+		err := startDebug(backend)
+		if err != nil {
```

Comment:
Why not `if err := startDebug(); err != nil`?

Reply:
Semantically no difference. Only preferring it on two lines for better readability. But can change.